packages/core/src/components/Table/Foot/Foot.styled.tsx

```diff
@@ -27,7 +39,6 @@ export const FootRow = styled('tr')`
     }
 
     td > ${Text.Style} {
-        line-height: 2rem;
-        font-size: 1.2rem;
+        ${getStyling}
```

Comment:
@mbetances805 You can use `getFontStyle`  like you have used above.

Reply:
Hi, when I use `getFontStyle` it includes the `font-weight`, which I do not want because it overrides the bold for the 'Viewing X of Y Results'

Reply:
Oh ok